front/src/applications/operationalStudies/components/SimulationResults/DriverTrainSchedule/DriverTrainScheduleStopList.tsx

```diff
@@ -48,7 +48,7 @@ export default function DriverTrainScheduleStopList({ train, baseOrEco }: Props)
           </thead>
           <tbody>
             {trainRegime.stops.map((stop, idx) => (
-              <DriverTrainScheduleStop stop={stop} idx={idx} train={train} />
+              <DriverTrainScheduleStop stop={stop} idx={idx} train={train} key={idx} />
```

Comment:
better avoid using the array indices unless you are absolutely sure the array is not going to be rearranged between two renders.
use unique
you can make one with the data if possible, like `${stop.id}-${stop.name}`